src/main/java/core/basesyntax/interfaces/DrawingFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface DrawingFigure {
+    void drawFigure();
```

Comment:
what if you are going to draw not only figure but other objects and using the same interface and method? it would be better to nor clarify what exactly you draw so you can rename interface to Drawable and method to draw.  It helps to implement polymorphism principle.

Reply:
But this task is about figures. Or do you mean we can use this method in another task in the future?